src/content/Home/hero_text.md

```diff
@@ -1 +1 @@
-One sentence description of how awesome HKN is in flowery language in three lines because aesthetics.
+Promoting excellence in the Illinois ECE community with the ideals of scholarship, character and attitude.
```

Comment:
Remove "the ideals of"

Reply:
+1, also if that's removed thoughts on changing "with" to "through"?

Reply:
Seems to be resolved - this text is no longer in our mission.